perl/lib/Gherkin/Line.pm

```diff
@@ -140,8 +139,8 @@ sub tags {
 
     my @tags;
     my @errors;
-    my @items = split( /@/, $items_line );
-    shift(@items);    # Blank first item
+    my @items = split /@/, $items_line;
+    shift @items;    # Blank first item
```

Comment:
```suggestion
    my (undef, @items) = split /@/, $items_line; # Drop blank first item
```

If we're going with idiomatic Perl, the way to throw away the leading element of an array is by putting "undef" at the beginning of the assignment list.